DxDispatch/src/dxdispatch/Device.cpp

```diff
@@ -364,6 +364,24 @@ std::vector<uint64_t> Device::ResolveTimestamps()
     return timestamps;
 }
 
+std::vector<double> Device::ResolveTimingSamples()
+{
+    std::vector<uint64_t> timestamps = ResolveTimestamps();
+
+    uint64_t frequency;
+    THROW_IF_FAILED(m_queue->GetTimestampFrequency(&frequency));
```

Comment:
Hi, I have a question, GetTimestampFrequency method is used to determine the rate at which the GPU timestamp counter increments, is it consisitent at any time, I mean between any two timestamps? Thus you can let each timestampDelta be divided by the same frequency?  Thanks!

Reply:
Yes, in D3D12 the timestamp frequency returned by `GetTimestampFrequency` is constant, even if other clock frequencies (e.g. shader cores boosted) change (see https://github.com/microsoft/DirectX-Specs/blob/master/d3d/CountersAndQueries.md#timestamp-frequency).